src/ontology/pato-edit.obo

```diff
@@ -22249,6 +22249,116 @@ comment: This is distinct from cortical bipolar morphology which has two dendrit
 is_a: PATO:0010006 ! cell morphology
 property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0001-7258-9596
 
+[Term]
+id: PATO:0070044
+name: cellular electrophysiological quality
+def: "A cellular quality that exists by virtue of the electrical properties of the cell." [WikipediaVersioned:Electrophysiology&oldid=1059385774]
+xref: ilxtr:ElectrophysiologicalPhenotype
```

Comment:
ilxtr is not in bioregistry - what is it?

Reply:
I've taken it straight from the link @tgbugs gave above (https://github.com/SciCrunch/NIF-Ontology/blob/78f17596d18bd3f1139ea33bbb6b112c97f8162f/ttl/phenotypes.ttl#L303) happy to change it to something else